FunctionalTableData/CellStyle.swift

```diff
@@ -39,15 +39,16 @@ public struct CellStyle {
 	public var selectionColor: UIColor? = CellStyle.selectionColor
 	/// The view's background color.
 	public var backgroundColor: UIColor?
-	/// The view that is displayed behind the cell’s other content.
-	public var backgroundView: UIView?
```

Comment:
can make this a computed property, getter returns result of backgroundViewProvider and setter wraps and creates a backgroundViewProvider

Reply:
setter? Should it not be deferred in a closure?